File: MemeLord/MemeLord/DataObjects/Response/CommentResponses/GetCommentsResponse.cs
using System.Collections.Generic;
using MemeLord.DataObjects.Dto;

namespace MemeLord.DataObjects.Response
{
    public class GetCommentsResponse
    {
        public int LastId { get; set; }
        public IEnumerable<CommentDto> CommentsList { get; set; }
    }
}

File: AngularApp/src/app/guest-site/dto/get-comments-response.ts
import { IMasterCommentDto} from './master-comment-dto'

export interface IGetCommentsResponse{
    lastId: number,
    commentsList: Array<IMasterCommentDto>
}

export class GetCommentsResponse implements IGetCommentsResponse {
    lastId: number;
    commentsList: Array<IMasterCommentDto>;

    constructor(lastId: number, commentsList: Array<IMasterCommentDto>){
        this.lastId = lastId;
        this.commentsList = commentsList;
    }
}